infrastructure/main.tf

```diff
@@ -58,15 +33,15 @@ module "api" {
     DRAFT_STORE_DB_NAME         = "draftstore"
     DRAFT_STORE_DB_CONN_OPTIONS = "${local.db_connection_options}"
 
-    IDAM_URL                    = "${var.idam_api_url}"
-    S2S_URL                     = "${local.s2s_url}"
+    IDAM_URL = "${var.idam_api_url}"
+    S2S_URL  = "${local.s2s_url}"
 
-    MAX_STALE_DAYS_DEFAULT      = "${var.max_stale_days_default}"
-    MAX_STALE_DAYS_CRON         = "${var.max_stale_days_cron}"
+    MAX_STALE_DAYS_DEFAULT = "${var.max_stale_days_default}"
+    MAX_STALE_DAYS_CRON    = "${var.max_stale_days_cron}"
 
-    FLYWAY_URL                  = "jdbc:postgresql://${var.db_host}:5432/draftstore${local.db_connection_options}"
-    FLYWAY_USER                 = "draftstore"
-    FLYWAY_PASSWORD             = "${data.azurerm_key_vault_secret.db_password.value}"
+    FLYWAY_URL      = "jdbc:postgresql://${var.db_host}:5432/draftstore${local.db_connection_options}"
+    FLYWAY_USER     = "draftstore"
+    FLYWAY_PASSWORD = "${data.azurerm_key_vault_secret.db_password.value}"
```

Comment:
oops, formatting gone. ah, not gone. there are single line separations around those blocks. as you were

Reply:
i ran `terraform fmt` over the file, changes weren't much so didn't separate to a different PR